tests/Doctrine/Tests/DBAL/Functional/TransactionTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Functional;
+
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\Tests\DbalFunctionalTestCase;
+use Throwable;
+use function in_array;
+use function sleep;
+
+class TransactionTest extends DbalFunctionalTestCase
+{
+    protected function setUp() : void
+    {
+        parent::setUp();
+
+        if (in_array($this->connection->getDatabasePlatform()->getName(), ['mysql'])) {
```

Comment:
Please use `instanceof` instead of the name for the better class discoverability. The `in_array()` seems redundant.

Reply:
Fixed.